common2/gratia/common2/pgpinput.py

```diff
@@ -10,10 +12,10 @@
     import uuid  # for unique cursor name
 except ImportError:
     # for python < 2.5 (uuid not available)
-    import uuid_replacement as uuid
+    from . import uuid_replacement as uuid
```

Comment:
If the comment above is accurate, then you can replace the whole try/except block with
```
import uuid
```

Reply:
Oh goodness you are right.